### PR TITLE
🎉  delivery optimisation (Option 234) testing 

### PR DESCRIPTION
**Pull request by mTouhid**
- update local kea config to include latest changes: client classes and option 234
 - update dhcp test client to kea 2.1 to obtain latest version of perfdhcp
 - include option 234 within the requested parameters list (option 55) using hex value
 
Later capture the packet of test client to inspect the DHCP offer packet.

🎉 Test result: tshark to read and check content of the DHCP offer packet and verify the option data.

Closes https://github.com/ministryofjustice/staff-device-dns-dhcp-admin/issues/513

### CODE REVIEW

**Review by ASTRobinson — APPROVED** [review]
LGTM

File: dhcp_test_client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+ENV GLIBC_VER=2.31-r0
+
+RUN apk add bash curl mysql mysql-client && \
+  curl -1sLf 'https://dl.cloudsmith.io/public/isc/kea-2-1/rsa.CC39698D5EDFF644.key' > /etc/apk/keys/kea-2-1@isc-CC39698D5EDFF644.rsa.pub && \
+  curl -1sLf 'https://dl.cloudsmith.io/public/isc/kea-2-1/config.alpine.txt?distro=alpine&codename=v3.13' >> /etc/apk/repositories && \
+  # curl -1sLf 'https://dl.cloudsmith.io/public/isc/kea-2-1/setup.alpine.sh' |  bash && \
+  apk update &&  \
+  apk upgrade && \
+  apk add build-base mysql-dev isc-kea-admin isc-kea-perfdhcp isc-kea-dhcp4 isc-kea-ctrl-agent isc-kea-hook-lease-cmds isc-kea-hook-stat-cmds isc-kea-hook-ha && \
```
Comment (emileswarts) [inline dhcp_test_client/Dockerfile:11]: Think we can remove some of these packages like:
isc-kea-hook-ha
isc-kea-hook-stat-cmds
etc

Not major, once we are able to deploy the new version of KEA, we should re-use that image.

File: dhcp_test_client/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM alpine:3.13
```
Comment (mTouhid) [inline dhcp_test_client/Dockerfile:1]: This new test container needs `RSpec` to run some feature tests. We need to use `ruby` base image.
cc: @emileswarts

File: dhcp_test_client/Dockerfile (hunk 1 above)
Comment (mTouhid) [reply dhcp_test_client/Dockerfile:11]: Thank you @emileswarts . Good point. I will push a new commit with the recommended changes.

Comment (mTouhid) [reply dhcp_test_client/Dockerfile]: Those packaged and some more have now been removed.

File: dhcp_test_client/Dockerfile (hunk 2 above)
Comment (mTouhid) [reply dhcp_test_client/Dockerfile]: Test container is now using ruby base image.

**Pull request merged**